stormpath/express-stormpath
# Fix session creation for auto-login when resetting password

Fixes so that a session cookie is created when `web.changePassword.autoLogin` is enabled and you're resetting the password for an account.

#### How to verify

1. Create a new application using [this gist](https://gist.github.com/typerandom/419d4346aa5e01ade26e50251768cbc9).
2. Start the application.
3. Navigate to the forgot password page (http://localhost:3000/forgot) and reset the password for one of your accounts.
4. Click the reset password link in the email you received and change the password for your account.
5. You should be redirected to `/` and logged in.

Fixes #164
